ivy/data_classes/array/array.py

```diff
@@ -1134,20 +1133,9 @@ def __irshift__(self, other):
         return ivy.bitwise_right_shift(self._data, other)
 
     def __deepcopy__(self, memodict={}):
-        try:
```

Comment:
I don't think we should be removing this, ideally we should always defer to the native framework dunder method (`__deepcopy__` in this case) as the default option, and should handle any framework-specific special cases in the `except` blocks.

Reply:
Yeah I had a feeling you weren't going to like that 😅